### PR TITLE
Add STACK v2.1.3.0

### PR DESCRIPTION
**Pull request by silkeh**
Add TransIP STACK.
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/stack.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+cask 'stack' do
+  version '2.1.3.0'
+  sha256 '21437c146af18b7865d691b26c5a0fb2282e1873b22fa08d0f702ed15d71ce15'
+
+  url "https://mirror.transip.net/stack/software/osx/stack-#{version}.pkg"
```
Comment (vitorgalvao) [inline Casks/stack.rb]: Where on the page is this link? We need to verify it.

Comment (adidalal) [inline Casks/stack.rb]: Looks like its from https://mirror.transip.net/stack/

Comment (silkeh) [inline Casks/stack.rb]: @adidalal is correct. That FAQ-page can be found on the homepage.

Comment (vitorgalvao) [inline Casks/stack.rb]: In that case, this needs a `# transip.net/stack was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

Comment (silkeh) [inline Casks/stack.rb]: I have added the above.

**Pull request merged**